test/vinyl/errinj_vylog.test.lua

```diff
@@ -198,7 +198,7 @@ _ = fiber.create(function() s:create_index('sk', {parts = {2, 'unsigned'}}) end)
 fiber.sleep(0.01)
 
 -- Should ignore the incomplete index on recovery.
-test_run:cmd('restart server default')
+test_run:cmd('restart server default with signal=KILL')
```

Comment:
Why?

Reply:
Added a comment here in the other places where we start using `KILL` signal.

<details>
<summary>diff</summary>

```diff
diff --git a/test/box-luatest/gh_8530_alter_space_snapshot_test.lua b/test/box-luatest/gh_8530_alter_space_snapshot_test.lua
index efe47814f..1e9eb0004 100644
--- a/test/box-luatest/gh_8530_alter_space_snapshot_test.lua
+++ b/test/box-luatest/gh_8530_alter_space_snapshot_test.lua
@@ -49,6 +49,8 @@ g.test_build_index = function(cg)
         box.snapshot()
         t.assert_equals(f:status(), 'suspended')
     end)
+    -- Use KILL because server will hang on shutdown due to injection.
+    -- We don't need graceful shutdown for the test anyway.
     cg.server.process:kill('KILL')
     cg.server:restart()
     cg.server:exec(function()
@@ -70,6 +72,8 @@ g.test_change_format = function(cg)
         box.snapshot()
         t.assert_equals(f:status(), 'suspended')
     end)
+    -- Use KILL because server will hang on shutdown due to injection.
+    -- We don't need graceful shutdown for the test anyway.
     cg.server.process:kill('KILL')
     cg.server:restart()
     cg.server:exec(function()
diff --git a/test/vinyl/errinj_vylog.result b/test/vinyl/errinj_vylog.result
index 90fc56d92..6ac76b2c8 100644
--- a/test/vinyl/errinj_vylog.result
+++ b/test/vinyl/errinj_vylog.result
@@ -399,6 +399,8 @@ fiber.sleep(0.01)
 ---
 ...
 -- Should ignore the incomplete index on recovery.
+-- Use KILL because server will hang on shutdown due to injection.
+-- We don't need graceful shutdown for the test anyway.
 test_run:cmd('restart server default with signal=KILL')
 s = box.space.test
 ---
diff --git a/test/vinyl/errinj_vylog.test.lua b/test/vinyl/errinj_vylog.test.lua
index 72513d923..54a69c659 100644
--- a/test/vinyl/errinj_vylog.test.lua
+++ b/test/vinyl/errinj_vylog.test.lua
@@ -198,6 +198,8 @@ _ = fiber.create(function() s:create_index('sk', {parts = {2, 'unsigned'}}) end)
 fiber.sleep(0.01)
 
 -- Should ignore the incomplete index on recovery.
+-- Use KILL because server will hang on shutdown due to injection.
+-- We don't need graceful shutdown for the test anyway.
 test_run:cmd('restart server default with signal=KILL')
 
 s = box.space.test

```

</details>